index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="first">
+      <nav class="menu">
+        <ul>
+          <li><a href="#">Logo</a></li>
+          <li><a href="#"><img src="./images/union.svg" alt="Hamburger logo"></a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="second">
+      <h1>I'm Mariane <br>
+        Glad to see you!
+      </h1>
+      <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h2>LET'S CONNECT</h2>
+      <div class="icons">
+        <img src="./images/twitter.svg" alt="Twitter logo">
+        <img src="./images/Linkedin.svg" alt="Linkedin logo">
+        <img src="./images/m.svg" alt="m logo">
+        <img src="./images/github.svg" alt="github logo">
+        <img src="./images/hands.svg" alt="hands logo">
+
+      </div>
```

Comment:
- [ ]  As this is a list of icons please use a `ul` tag to group them, and use a `li` tag for each icon. This will make your code more semantic. :+1: :sunglasses: 
 - [ ]  Also, make sure that each icon has a link (`a` tag) because each icon is supposed to redirect to your own social media.